.github/workflows/oxcaml.yml

```diff
@@ -29,7 +29,7 @@ jobs:
 
       - name: Install deps
         run: |
-          opam install . --deps-only
+          opam install . --deps-only spawn
```

Comment:
Annoyingly, this will install the deps of spawn and not spawn itself. There is no way to do this in a single command AFAIK. Best is to do another `opam install spawn` straight after instead.